src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,97 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  items: string[],
+  pageAmount: number,
+  currentPage: number,
+  onPageChange: React.MouseEventHandler<HTMLAnchorElement>,
+};
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  pageAmount,
+  currentPage,
+  onPageChange,
+}) => {
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={classNames(
+            'page-item',
+            {
+              disabled: currentPage === 1,
+            },
+          )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={(event) => onPageChange(event)}
+          >
+            «
+          </a>
+        </li>
+
+        {getNumbers(1, pageAmount).map(page => {
+          const isActive = page === currentPage;
+
+          return (
+            <li
+              className={classNames(
+                'page-item',
+                {
+                  active: isActive,
+                },
+              )}
+              key={page}
+            >
+              <a
+                data-cy="pageLink"
+                className="page-link"
+                href={`#${page}`}
+                onClick={(event) => {
+                  if (!isActive) {
+                    onPageChange(event);
+                  }
+                }}
```

Comment:
if you use some conditions - this should be a sign for you that you should move it to a separate method